### PR TITLE
Fix signatures for classmethods in several builtins classes

### PR DESCRIPTION
**Pull request by AlexWaygood**
Closes https://github.com/python/mypy/issues/11922, & makes similar changes to other classmethods in builtins classes where the signatures are currently problematic.

### CODE REVIEW

File: stdlib/builtins.pyi
Comment (Akuli) [inline stdlib/builtins.pyi:214]: The comment should be moved to the line that defines the `bytes` argument.

**Review by Akuli — APPROVED** [review]

**Comment by Akuli** [comment]
Somehow I clicked approve, when I meant to leave a comment without approval. I will merge once you have moved the comment :)

File: stdlib/builtins.pyi
Comment (AlexWaygood) [reply stdlib/builtins.pyi]: Done! :)

**Pull request merged**
